hack/lib.sh

```diff
@@ -153,18 +153,21 @@ function storage_admin_set_up() {
   local project_id=${1}
   local pubsub_service_account=${2}
   local pubsub_service_account_key_temp=${3}
```

Comment:
Looks like this param is not needed any more?

Reply:
Good catch, removed.